### DISCUSSION TITLE
Code review bot - CodeRabbit

### DISCUSSION THREAD

**Discussion by v1gnesh** [zopencommunity/meta#840] [Ideas]
https://coderabbit.ai/

The site is pretty self-explanatory. The Pro version is free for OSS projects.

**Comment by IgorTodorovskiIBM** [comment]
I like that you can implement it as a reviewer "AI-first pull request reviewer with context-aware feedback, line-by-line code suggestions, and real-time chat." I think it will help save time. @MikeFultonDev, your thoughts?

**Reply by v1gnesh** [reply]
Could we do this :) ?

**Reply by IgorTodorovskiIBM** [reply]
@DevonianTeuchter @HarithaIBM @MikeFultonDev @AnthonyGiorgio  thoughts? This could help automate some of the PR reviewing, help summarize the PRs...
Here's an example PR with CodeRabbit integrated: https://github.com/Sinaptik-AI/pandas-ai/pull/713

**Reply by AnthonyGiorgio** [reply]
I guess it's okay?  We can always ignore it, or disable it if it turns out to be useless.
